src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
```

Comment:
- [ ] Please work on times events currently when I try let say `8*8` I am getting an error instead of the result
![er](https://user-images.githubusercontent.com/44745855/133482219-9e1c09d8-9fe6-4a6c-8561-8253d4e5129a.png)

Please fix that